### PR TITLE
fix: humoruniv.com (naver powerlink)

### PR DESCRIPTION
**Pull request by seia-soto**
https://github.com/AdguardTeam/AdguardFilters/issues/116329

### CODE REVIEW

**Comment by seia-soto** [comment]
Cosmetic rule is for non script-injectable users.

File: filters-AG/scriptlets.txt [hunk 1]
```diff
@@ -1,6 +1,13 @@
 ! PopMagic
 watchfreejavonline.co#%#//scriptlet("abort-current-inline-script", "popMagic.init")
 !
+humoruniv.com#%#//scriptlet("set", "ch2", "noopFunc")
+humoruniv.com#%#//scriptlet("set", "ch2_submit", "noopFunc")
+humoruniv.com#%#//scriptlet("set", "ch4", "noopFunc")
+humoruniv.com#%#//scriptlet("set", "ch4_submit", "noopFunc")
+humoruniv.com#%#//scriptlet("set", "naver_adpost_ch1", "noopFunc")
+humoruniv.com#%#//scriptlet("set", "naver_adpost_ch2_backfill", "noopFunc")
+humoruniv.com#%#//scriptlet("set", "show_adpost", "noopFunc")
```
Comment (piquark6046) [inline filters-AG/scriptlets.txt]: Please modify from `set` to `set-constant`.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Are `m.humoruniv.com##div[class*="_ad"][style]:not([id])` and `m.humoruniv.com##body form[id][name][action="boardexp/styleofpower.html"] + div[id][style]` still valid?

**Comment by seia-soto** [comment]
No, they're deprecated. (Nothing matched)

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**